### PR TITLE
[MM-18579] add transition to about, team members, settings modal closing

### PR DESCRIPTION
**Pull request by Willyfrog**
#### Summary
Add animatin transition  when hiding the following modals:
- about
- team settings
- manage team members

#### Ticket Link

[MM-18579](https://mattermost.atlassian.net/browse/MM-18579)

### CODE REVIEW

**Comment by thekiiingbob** [comment]
@Willyfrog Checked out the deployed server and looks good to me. Could you add a test for team_settings_modal? That looks to be the only one without one, unless there's a reason why not to.

**Comment by Willyfrog** [comment]
no, the only reason is that I only modified the failing ones, but having an extra one is not a bad idea. Thanks!

File: components/team_members_modal/team_members_modal.jsx [hunk 1]
```diff
@@ -10,19 +10,30 @@ import MemberListTeam from 'components/member_list_team';
...
         if (this.props.onLoad) {
             this.props.onLoad();
         }
     }
 
     onHide = () => {
+        this.setState({show: false});
+    }
+
+    onExit = () => {
```
Comment (saturninoabril) [inline components/team_members_modal/team_members_modal.jsx]: I don't see this used elsewhere.

File: components/about_build_modal/about_build_modal.jsx [hunk 2]
```diff
@@ -42,10 +37,29 @@ export default class AboutBuildModal extends React.PureComponent {
...
+
     doHide = () => {
+        this.setState({show: false});
+    }
+
+    handleExit = () => {
         this.props.onHide();
     }
 
+    handleToggle = () => {
```
Comment (saturninoabril) [inline components/about_build_modal/about_build_modal.jsx]: This is not used.

**Pull request merged**